src/common/indexer/elastic/elastic.indexer.service.ts

```diff
@@ -446,6 +446,22 @@ export class ElasticIndexerService implements IndexerInterface {
     return await this.elasticService.getList('scdeploys', "contract", elasticQuery);
   }
 
+  async getAccountContracts(pagination: QueryPagination, address: string): Promise<any[]> {
```

Comment:
could have used the same function and parameterize the es field to be used for query match (`fieldToMatch` that can be either `currentOwner`, either `deployer`)

Reply:
i will leave for now the actual implementation, more refactors for elastic indexer will follow